server.js

```diff
@@ -4,12 +4,13 @@
 // 3rd part libraries
 const express = require('express');
 const app = express();
-// require('dotenv').config();
+const logger = require('/middleware/logger');
```

Comment:
One thing causing an issue, this needs to be:
`const logger = require('./middleware/logger')`